nucypher/policy/conditions/offchain.py

```diff
@@ -148,6 +121,55 @@ def query_response(self, data: Any) -> Any:
 
         return result
 
+
+class JsonApiCondition(BaseExecAccessControlCondition):
+    """
+    A JSON API condition is a condition that can be evaluated by reading from a JSON
+    HTTPS endpoint. The response must return an HTTP 200 with valid JSON in the response body.
+    The response will be deserialized as JSON and parsed using jsonpath.
+    """
```

Comment:
You can use the logic of sequential conditions to mitigate the issue with unpredictably deprecated APIs. You probably still need updatable conditions eventually

Reply:
Interesting thought. Could you expand on that for me?